zmon_cli/client.py

```diff
@@ -56,16 +62,31 @@ def wrapper(*args, **kwargs):
         try:
             return f(*args, **kwargs)
         except:
-            logger.error('Zmon client failed in: {}'.format(f.__name__))
+            logger.error('ZMON client failed in: {}'.format(f.__name__))
             raise
 
     return wrapper
 
 
 def compare_entities(e1, e2):
-    e1.pop('last_modified', None)
-    e2.pop('last_modified', None)
-    return json.loads(json.dumps(e1)) == json.loads(json.dumps(e2))
+    e1_copy = e1
+    e2_copy = e2
+
+    if isinstance(e1, dict):
```

Comment:
can e1/e2 be something else than a dict? I guess not..

(so I would get rid of the `if`)